using-timescaledb/hypertables.md

```diff
@@ -106,55 +115,56 @@ corresponds to increased planning latency for some types of queries.
-query parallelization only works on a single physical table (and thus
-a single chunk). We might add our own support for this, but it is not
-currently supported.
-
-Thus, users looking for parallel I/O have two options:
-
-1. Use a RAID setup across multiple physical disks, and expose a
-single logical disk to the hypertable (i.e., via a single tablespace).
-
-1. For each physical disk, add a separate tablespace to the
-database.  TimescaleDB allows you to actually add multiple tablespaces
-to a *single* hypertable (although under the covers, each underlying
-chunk will be mapped by TimescaleDB to a single tablespace / physical
-disk).
-
-We recommend a RAID setup when possible, as it supports both forms of
-parallelization described above (i.e., separate queries to separate
-disks, single query to multiple disks in parallel).  The multiple
-tablespace approach only supports the former.  With a RAID setup,
-*no spatial partitioning is required*.
-
-That said, when using space partitions, we recommend using 1
-space partition per disk.
-
-TimescaleDB does *not* benefit from a very large number of space
-partitions (such as the number of unique items you expect in partition
-field).  A very large number of such partitions leads both to poorer
-per-partition load balancing (the mapping of items to partitions using
-hashing), as well as much increased planning latency for some types of
-queries.
+N buckets. In a distributed hypertable, each bucket of the primary
+space dimension corresponds to a specific data node (although two or
+more buckets could map to the same node). In non-distributed
+hypertables, each bucket can map to a distinct disk (using, e.g., a
+tablespace).
+
+>:TIP: TimescaleDB does *not* benefit from a very large number of
+space partitions (such as the number of unique items you expect in
+partition field).  A very large number of such partitions leads both
+to poorer per-partition load balancing (the mapping of items to
+partitions using hashing), as well as increased planning latency
+for some types of queries. We recommend tying the number of space
+partitions to the number of disks and/or data nodes.
+
+Spreading chunks along disks and nodes in the space dimension allows
+for increased I/O parallelization, either by (a) having multiple
+concurrent client processes, or, by (b) splitting the work of a single
+client across multiple worker processes on a single node or multiple
+concurrent requests across several data nodes.
+
+In summary, to benefit from parallel I/O, on can do one of the
+following:
+
+1. For each physical disk on a single instance, add a separate
+tablespace to the database.  TimescaleDB allows you to actually add
+multiple tablespaces to a *single* hypertable (although under the
+covers, each underlying chunk will be mapped by TimescaleDB to a
+single tablespace / physical disk).
+
+1. Configure a distributed hypertable that spreads inserts and queries
```

Comment:
```suggestion
2. Configure a distributed hypertable that spreads inserts and queries
```
I doubt the number will be automatically updated by markdown due to new line in between.